imagersite/imager_images/tests/test_models.py

```diff
@@ -0,0 +1,135 @@
+from django.test import TestCase
+# from django.db.models.fields.files import ImageFieldFile
+from imager_images.models import Photo, Album
+from imager_profile.tests import UserFactory
+from datetime import timezone, datetime
+import factory
+
+
+class AlbumFactory(factory.django.DjangoModelFactory):
+    """Create Test Albums Factory."""
+
+    class Meta():
+        """Model is Album."""
+
+        model = Album
+
+
+class PhotoFactory(factory.django.DjangoModelFactory):
+    """Create Test Photo Factory."""
+
+    class Meta():
+        """Model is photo."""
+
+        model = Photo
+
+
+class SingleAlbumAndPhotoTestCase(TestCase):
+    """Asserts that a single User's Photos and Albums work as expected."""
+
+    def setUp(self):
+        """Set Up Testing Environment."""
+        self.maithrika = UserFactory.create(
+            username="maithrika"
+        )
+        self.vacation = AlbumFactory.create(
+            user=self.maithrika,
+            title="Vacation",
+            description="Album of pics",
+            published="public"
+        )
+        self.selfie = PhotoFactory.create(
+            title="selfie",
+            user=self.maithrika,
+            description="just a picture",
+            published="public"
+        )
+
+    def test_user_in_db(self):
+        """Assert User has primary key."""
+        self.assertTrue(self.maithrika.pk)
+
+    def test_photo_in_db(self):
+        """Assert User has primary key."""
+        self.assertTrue(self.selfie.pk)
+
+    def test_album_in_db(self):
+        """Assert User has primary key."""
+        self.assertTrue(self.vacation.pk)
+
+    def test_album_attributes(self):
+        """Assert Album has necessary attributes."""
+        album = self.vacation
+        self.assertEqual(album.user, self.maithrika)
+        self.assertEqual(album.title, "Vacation")
+        self.assertEqual(album.description, "Album of pics")
+        self.assertEqual(album.published, "public")
+        self.assertIsNone(album.date_published)
+
+    def test_album_datetimes(self):
+        """Assert album's datetime fields are datetime."""
+        self.assertGreater(datetime.now(timezone.utc), self.vacation.date_uploaded)
+        self.assertGreater(datetime.now(timezone.utc), self.vacation.date_modified)
```

Comment:
It's worth thinking about better ways to test time values.  One of my favorites is to assert that the delta between when something was created and when you are testing it is below a certain value.  You can read more about testing time in [this blog post by Ned Batchelder](http://nedbatchelder.com/blog/201504/testing_time.html)